dev/src/components/payments/AfterpayPaymentConfig.vue

```diff
@@ -53,17 +53,25 @@
         </div>
       </div>
     </div>
+    <FinancialWarning @set-warning="setWarning" :enabled="config.financial_warning" />
 </template>
 
 <script>
-import { inject } from 'vue'
+import {inject} from 'vue'
 import {useI18n} from "vue-i18n";
-
+import FinancialWarning from "../fields/FinancialWarning.vue";
 export default {
     name: "AfterpayPaymentConfig",
+    components: {
+        FinancialWarning
+    },
+    methods: {
```

Comment:
why don't you set the `financial_warning` value in the `FinancialWarning` component?